src/calibration/measurement_export.cpp

```diff
@@ -0,0 +1,230 @@
+        export_to_csv(op->input_right());
+
+      }
+    }
+
+    void MeasurementExport::_export_typed_operator(std::shared_ptr<const AbstractOperator> op) const {
+      switch(op->type()){
+        case OperatorType::Aggregate:           _export_generic(op);    break;
+        case OperatorType::Alias:               _export_generic(op);    break;
+        case OperatorType::Delete:              _export_generic(op);    break;
+        case OperatorType::Difference:          _export_generic(op);    break;
+        case OperatorType::ExportBinary:        _export_generic(op);    break;
+        case OperatorType::ExportCsv:           _export_generic(op);    break;
+        case OperatorType::GetTable:            _export_generic(op);    break;
+        case OperatorType::ImportBinary:        _export_generic(op);    break;
+        case OperatorType::ImportCsv:           _export_generic(op);    break;
+        case OperatorType::IndexScan:           _export_generic(op);    break;
+        case OperatorType::Insert:              _export_generic(op);    break;
+        case OperatorType::JoinHash:            _export_generic(op);    break;
+        case OperatorType::JoinIndex:           _export_generic(op);    break;
+        case OperatorType::JoinNestedLoop:      _export_generic(op);    break;
+        case OperatorType::JoinSortMerge:       _export_generic(op);    break;
+        case OperatorType::JoinVerification:    _export_generic(op);    break;
+        case OperatorType::Limit:               _export_generic(op);    break;
+        case OperatorType::Print:               _export_generic(op);    break;
+        case OperatorType::Product:             _export_generic(op);    break;
+        case OperatorType::Projection:          _export_generic(op);    break;
+        case OperatorType::Sort:                _export_generic(op);    break;
+        case OperatorType::TableScan:           _export_table_scan(op); break;
+        case OperatorType::TableWrapper:        _export_generic(op);    break;
+        case OperatorType::UnionAll:            _export_generic(op);    break;
+        case OperatorType::UnionPositions:      _export_generic(op);    break;
+        case OperatorType::Update:              _export_generic(op);    break;
+        case OperatorType::Validate:            _export_generic(op);    break;
+        case OperatorType::CreateTable:         _export_generic(op);    break;
+        case OperatorType::CreatePreparedPlan:  _export_generic(op);    break;
+        case OperatorType::CreateView:          _export_generic(op);    break;
+        case OperatorType::DropTable:           _export_generic(op);    break;
+        case OperatorType::DropView:            _export_generic(op);    break;
+        case OperatorType::Mock:                _export_generic(op);    break;
+      }
+    }
+
+    //TODO Construct path before starting export-> save it in map
+    std::string MeasurementExport::_path(OperatorType operator_type) const {
```

Comment:
Measurement is to broad for my taste. `Calibration`? `CalibrationMeasurement`?

Please consider that this is a rather narrow area of Hyrise that you are working on here. Your names should not confuse other students which use "core Hyrise".